fastlane/docs/Codesigning/XcodeProject.md

```diff
@@ -31,6 +31,18 @@ PROVISIONING_PROFILE_SPECIFIER = "[team_id]/[profile_name]";
 
 You could pass the profile specifier via environment variables too (See Xcode 7 approach below), however in most cases it's enough to define it statically in your project, per target, since usually neither your Team ID, nor the provisioning profile change.
 
+If you want to set it dynamically, you can use sigh or match to fill the environment variables:
```

Comment:
Interesting. Why doesn't match pre-fill the environment so that you don't have to set it again manually as a user?